timetracker/entries/views.py

```diff
@@ -1,31 +1,23 @@
 from django.shortcuts import render, get_object_or_404, redirect
+from django.views.generic import CreateView, RedirectView
+from django.core.urlresolvers import reverse
 
 from .forms import EntryForm, ProjectForm, ClientForm
 from .models import Client, Entry, Project
 
 
-def clients(request):
-    if request.method == 'POST':
-        # Create our form object with our POST data
-        form = ClientForm(request.POST)
-        if form.is_valid():
-            # If the form is valid, create a client with submitted data
-            # Below is the shortcut equivalent of:
-            # client = Client()
-            # client.name = form.cleaned_data['name']
-            # client.save()
-            # Sometimes you don't want to save the object until the end,
-            # sometimes you don't care!
-            client = Client.objects.create(name=form.cleaned_data['name'])
-            return redirect('client-list')
-    else:
-        form = ClientForm()
+class ClientCreateView(CreateView):
+    model = Client
+    template_name = 'clients.html'
+    form_class = ClientForm
 
-    client_list = Client.objects.all()
-    return render(request, 'clients.html', {
-        'client_list': client_list,
-        'form': form,
-    })
+    def get_success_url(self):
+        return reverse('client-list')
```

Comment:
You can use `success_url = reverse_lazy('client-list')` instead ;) (`reverse_lazy is at the same place as `reverse`)